django-stubs/urls/conf.pyi

```diff
@@ -11,9 +11,7 @@ from ..http.response import HttpResponseBase
 
 _URLConf: TypeAlias = str | ModuleType | Sequence[_AnyURL]
 
-def include(
-    arg: _URLConf | tuple[_URLConf, str], namespace: str | None = ...
-) -> tuple[Sequence[URLResolver | URLPattern], str | None, str | None]: ...
+def include(arg: _URLConf | tuple[_URLConf, str], namespace: str | None = ...) -> IncludedURLConf: ...
```

Comment:
Maybe we can:
- rename it to `_IncludedURLConf`
or
- add it to `ext` module and import from there?

Reply:
Perhaps prefix with an underscore to _`IncludedURLConf`? Thinking since it's only an alias

Reply:
Done